### PR TITLE
Hellbot printers definitions added

### PR DESCRIPTION
**Pull request by F-Fischer**

### CODE REVIEW

**Comment by diegopradogesto** [comment]
Devs. CURA-6813

File: resources/definitions/hellbot_magna_I.def.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ 		"machine_extruder_trains": {
+ 			"0": "hellbot_magna_i_extruder"
+ 		}
+ 	},
+ 	"overrides": {
+ 		"machine_name": {
+ 			"default_value": "Hellbot Magna I"
+ 		},
+ 		"machine_width": {
+ 			"default_value": 230
```
Comment (diegopradogesto) [inline resources/definitions/hellbot_magna_I.def.json]: The specifications indicates that the X dimension is 220mm: https://hellbot.xyz/magna-1/
```suggestion
 			"default_value": 220
```

File: resources/definitions/hellbot_magna_dual.def.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+ {
+ 	"version": 2,
+ 	"name": "Hellbot Magna dual",
```
Comment (diegopradogesto) [inline resources/definitions/hellbot_magna_dual.def.json]: The name in the website is with capital letters.
```suggestion
 	"name": "Hellbot Magna DUAL",
```

File: resources/definitions/hellbot_magna_dual.def.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ 		"platform_offset": [0, -1, 0],
+ 		"has_materials": true,
+ 		"machine_extruder_trains": {
+ 			"0": "hellbot_magna_dual_extruder_1",
+ 			"1": "hellbot_magna_dual_extruder_2"
+ 		}
+ 	},
+ 	"overrides": {
+ 		"machine_name": {
+ 			"default_value": "Hellbot Magna I"
```
Comment (diegopradogesto) [inline resources/definitions/hellbot_magna_dual.def.json]: Name here is inconsistent, maybe a copy-paste mistake.
```suggestion
 			"default_value": "Hellbot Magna DUAL"
```

File: resources/extruders/hellbot_magna_dual_extruder_2.def.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
+ {
+    "id": "hellbot_magna_dual_extruder_2",
+    "version": 2,
+    "name": "Extruder 2",
+    "inherits": "fdmextruder",
+    "metadata": {
+        "machine": "hellbot_magna_dual",
+        "position": "0"
```
Comment (diegopradogesto) [inline resources/extruders/hellbot_magna_dual_extruder_2.def.json]: This should be position `1`
```suggestion
        "position": "1"
```

**Comment by F-Fischer** [comment]
All requested changes commited

**Comment by Ghostkeeper** [comment]
Thanks for the profiles! Looks good for the most part. I'm changing the tabs to spaces if you don't mind. It's a bit more consistent with the rest then. The profiles themselves look fine now.

**Comment by Ghostkeeper** [comment]
I had to change one more thing to fix the dual extrusion of the Magna Dual printer: 0161a1eca5e177fe876fb879111b8ad14aacf89d

**Pull request merged**